packages/twenty-server/src/workspace/messaging/services/gmail-partial-sync.service.ts

```diff
@@ -231,4 +251,34 @@ export class GmailPartialSyncService {
       },
     );
   }
+
+  private isMessageBlocked = (
```

Comment:
```suggestion
  private isHandleBlocked = (
```
It seems more appropriate?